providers/implementations/keymgmt/ml_kem_kmgmt.c

```diff
@@ -396,6 +432,11 @@ static int ml_kem_get_params(void *vkey, OSSL_PARAM params[])
     ML_KEM_KEY *key = vkey;
     const ML_KEM_VINFO *v = ossl_ml_kem_key_vinfo(key);
     OSSL_PARAM *p;
+    const char *pubparams[] = {
+        OSSL_PKEY_PARAM_PUB_KEY,
+        OSSL_PKEY_PARAM_ENCODED_PUBLIC_KEY
```

Comment:
Is this even needed to be supported?

Reply:
Yes, EVP has methods for getting and setting the encoded public key needed for key agreement.